xonsh/xonfig.py

```diff
@@ -81,8 +81,8 @@
 
                       {{BOLD_WHITE}}Foreign Shell Setup{{RESET}}
                       {{YELLOW}}-------------------{{RESET}}
-The xonsh shell has the ability to interface with foreign shells such
-as Bash, or zsh (fish not yet implemented).
+The xonsh shell has the ability to interface with foreign shells:
+Bash or zsh. The other shells can be implemented as well.
```

Comment:
```suggestion
The xonsh shell has the ability to interface with Bash or zsh
via the foreign shell interface.
```

Maybe something like this?  We mention the foreign shell interface but make no promises about adding new extensions.

Reply:
Sounds good!